cddl_gen/__init__.py

```diff
@@ -5,7 +5,15 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+from pathlib import Path
+
 from .cddl_gen import (
     CddlValidationError,
     DataTranslator,
 )
+
+script_path = Path(__file__).absolute().parents[0]
+VERSION_path = Path(script_path, "VERSION")
+
+with open(VERSION_path, 'r') as f:
```

Comment:
Minor: since `VERSION_path` is already a `pathlib.Path` object, you can also use its `open` method, which defaults to `mode='r'`:

```
with VERSION_path.open() as f:
    ....
```

Not a blocker to approval though.